openmdao/drivers/pyoptsparse_driver.py

```diff
@@ -629,7 +630,8 @@ def run(self):
         # framework is left in the right final state
         dv_dict = sol.getDVs()
         for name in self._designvars:
-            self.set_design_var(name, dv_dict[model._get_prom_name(name)])
+            # self.set_design_var(name, dv_dict[model._get_prom_name(name)])
```

Comment:
could clean up some old commented code in this file

Reply:
done